engine/gamesys/src/gamesys/scripts/script_factory.cpp

```diff
@@ -235,16 +237,51 @@ namespace dmGameSystem
      * end
      * ```
      */
+    static int FactoryComp_CreateWithMessage(lua_State* L, dmGameObject::HCollection collection, dmGameObject::HInstance sender_instance, dmMessage::URL* receiver,
```

Comment:
This is the more surprising case: When does this actually happen?
I.e. we call factory.create, but _don't_ have a game object instance set?

I left it "as-is" for now. Potential candidate for removal imho.